src/test/java/com/softserveinc/ita/mmakoviichuk/Test.java

```diff
@@ -0,0 +1,44 @@
+package com.softserveinc.ita.mmakoviichuk;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.*;
+
+import java.util.List;
+
+public class Test {
+    WebDriver driver;
+    @BeforeSuite
+    public void beforeSuite() {
+        System.setProperty("webdriver.chrome.driver",
+                "C:\\Users\\Ferrl\\IdeaProjects\\chromedriver_win32\\chromedriver.exe");
+
```

Comment:
delete empty line, place a chromedriver in this package 'main/java/com/softserveinc/ita' create here a package 'drivers'